src/main/java/com/weaverplatform/postgresql/controllers/NodeController.java

```
@@ -37,10 +37,15 @@
     String database = req.queryParams("database");
     String creator  = req.queryParams("creator");
 
-    String writeOperationsJson = req.body();
-    List<WriteOperation> writeOperations = WriteOperationParser.parse(writeOperationsJson, creator);
+    try {
+      String writeOperationsJson = req.body();
+      List<WriteOperation> writeOperations = WriteOperationParser.parse(writeOperationsJson, creator);
 
-    Postgres.INSTANCE.getDatabase(database).write(writeOperations);
+      Postgres.INSTANCE.getDatabase(database).write(writeOperations);
+    } catch(Exception e) {
+      logger.error(e.getLocalizedMessage(), e);
+      return "Failure";
```

Returning a string here results in the http status code being set to 200, which it should not be. Maybe just a `throw new RuntimeException("Failure")` is enough if the point of this catch is to log errors and swallow any additional details to prevent leaking them to the user.

Also: doesn't other code rely on this error?